src/app/ngrx/effects/castle-catalog.effects.spec.ts

```diff
@@ -1,10 +1,9 @@
 import { Actions } from '@ngrx/effects';
 import { TestBed, async } from '@angular/core/testing';
 import { cold, hot } from 'jasmine-marbles';
-import { StoreModule, Store } from '@ngrx/store';
+import { StoreModule, Store, Action } from '@ngrx/store';
 import { EffectsModule } from '@ngrx/effects';
-import { getActions, TestActions } from './test.actions';
-
+import { provideMockActions } from '@ngrx/effects/testing';
```

Comment:
ngrx adds `provideMockActions` for testing effects https://ngrx.io/guide/effects/testing